src/deploy/nsfs.service

```diff
@@ -1,13 +1,14 @@
 [Unit]
 Description=The NooBaa nsfs service.
+After=gpfs-wait-mount.service
 
 [Service]
 Restart=always
 RestartSec=2
 User=root
 Group=root
-Environment=
 ExecStart=/usr/local/noobaa-core/bin/node /usr/local/noobaa-core/src/cmd/nsfs.js
+EnvironmentFile=/etc/noobaa.conf.d/.env
```

Comment:
We already have support for `$NOOBAA_DIR/.env` file though. Should we document somewhere that these two `.env` files are different and that `$NOOBAA_DIR/.env` will always supersede `/etc/noobaa.conf.d/.env` one?